python/ray/data/tests/test_all_to_all.py

```diff
@@ -208,6 +214,44 @@ def create_large_data(group):
     ds.take(1)
 
 
+@pytest.mark.parametrize("keys", ["A", ["A", "B"]])
+def test_agg_inputs(ray_start_regular_shared, keys):
+    xs = list(range(100))
+    ds = ray.data.from_items([{"A": (x % 3), "B": x, "C": (x % 2)} for x in xs])
```

Comment:
Please add the test with None values (like in the original issue)

Reply:
Let's also make sure we cover this part https://github.com/ray-project/ray/pull/48697#discussion_r1840974236